test/arrays/ferguson/semantic-examples/0-init-alias.chpl

```diff
@@ -0,0 +1,10 @@
+use samples;
+// var one = new R(1);
```

Comment:
OK. I'm updating the comments to say e.g.

      // one defined in samples as new R(1)